Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	1	create_and_make_expr_table
	2

[Fri Aug 24 17:10:43 2018]
rule create_and_make_expr_table:
    input: tmp/rpg_dict
    output: /net/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/results/expr.tab
    jobid: 2

    [Fri Aug 24 17:10:44 2018]
    Error in rule create_and_make_expr_table:
        jobid: 2
        output: /net/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/results/expr.tab

RuleException:
CalledProcessError in line 253 of /net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/CREATE_EXPR_TABLE.smk:
Command ' set -euo pipefail;  Rscript /net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/lib/.snakemake.cdybss79.collect_rpg_data.R ' returned non-zero exit status 1.
  File "/net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/CREATE_EXPR_TABLE.smk", line 253, in __rule_create_and_make_expr_table
  File "/home/thkuo/miniconda3/envs/seq2geno/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v5/.snakemake/log/2018-08-24T171036.358506.snakemake.log
